__tests__/index.test.js

```diff
@@ -35,6 +35,48 @@ describe("/api", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET 200: Responds with the appropriate article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body;
+        expect(article).toMatchObject({
+          author: "butter_bridge",
+          title: "Living in the shadow of a great man",
+          article_id: 1,
+          body: "I find this existence challenging",
+          topic: "mitch",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
```

Comment:
This is a brilliant test! There is one thing that is out of place, and that is line `44`, you aren't destructuring there so it might be that the response isn't being sent on a key of `article` because I would expect it to be `const { article } = body;`